app.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "collections",
+  "repository": "https://github.com/alphagov/collections",
+  "env": {
+    "GOVUK_APP_DOMAIN": {
+      "value": "www.gov.uk"
+    },
+    "GOVUK_ASSET_ROOT": {
+      "value": "https://govuk-collections.herokuapp.com"
```

Comment:
I think this needs to point to the review app itself? Otherwise you'll get the assets from master instead of the PR?